### ISSUE TITLE
Questions about the dataset

### ISSUE THREAD

**Issue by CodeNinjaja** [yzhHoward/SMART#3]
Thank you for the great work, and I appreciate your open-source contributions. However, I found some confusing aspects while reviewing the source code. 

Firstly, you use the training, validation, and test sets to compute the mean and standard deviation in the data standardization process. Doesn't this risk information leakage? The mean and standard deviation should typically be calculated from the training set.

Secondly, in your code, when splitting the dataset, different random seeds result in different splits of the dataset. Is this reasonable? Shouldn't the data splits be kept consistent, and then the model performance be evaluated across different random seeds to compute the mean and standard deviation?

**Comment by yzhHoward** [comment]
You are right. It is better to use a fixed seed to divide the dataset and calculate the mean and standard deviation on the training set. I overlooked this point when writing this paper. In addition, using a new seed to divide the dataset each time will cause too much performance fluctuation. It is recommended that you use a fixed seed.
